bitnami/fluent-bit/Chart.yaml

```diff
@@ -28,4 +28,4 @@ maintainers:
 name: fluent-bit
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/fluent-bit
-version: 1.0.1
+version: 1.1.1
```

Comment:
Incorrect bump! `1.0.1` should bump a minor to `1.1.0`

Reply:
Sorry fix this asap